pyproject.toml

```diff
@@ -48,7 +47,7 @@ exclude = ["tests", "tests.*"]
 Home-page = "https://github.com/pypa/readme_renderer"
 
 [build-system]
-requires = ["setuptools>=40.8.0"]
+requires = ["setuptools>=77.0.1"]
```

Comment:
I would suggest `"setuptools>=77.0.3"` for consistency with the [Python Packaging User Guide](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files) or `"setuptools>=77.0.0"` for consistency with the [Setuptools documentation](https://setuptools.pypa.io/en/stable/userguide/pyproject_config.html).

I understand that in theory 77.0.0 supports PEP 639, but in practice 77.0.3 fixes known early bugs of the PEP 639 implementation. But then 77.0.1 might be good enough for this specific use case.

Reply:
I've updated to require `setuptools>=77.0.3`. Thanks for suggesting an update that resolves some known bugs!